Biasing/src/PhotoNuclearXsecBiasingOperator.cxx

```diff
@@ -0,0 +1,97 @@
+/**
+ * @file XsecBiasingPlugin.cxx
+ * @brief Geant4 Biasing Operator used to bias the occurence of photonuclear 
+ *        events by modifying the cross-section.
+ * @author Omar Moreno
+ *         SLAC National Accelerator Laboratory
+ */
+
+#include "Biasing/PhotoNuclearXsecBiasingOperator.h"
+
+namespace ldmx { 
+
+    const std::string PhotoNuclearXsecBiasingOperator::PHOTONUCLEAR_PROCESS = "photonNuclear"; 
+    
+    const std::string PhotoNuclearXsecBiasingOperator::CONVERSION_PROCESS = "conv";  
+
+    PhotoNuclearXsecBiasingOperator::PhotoNuclearXsecBiasingOperator(std::string name) :
+        XsecBiasingOperator(name) {
+    }
+
+    PhotoNuclearXsecBiasingOperator::~PhotoNuclearXsecBiasingOperator() {
+    }
+
+    G4VBiasingOperation* PhotoNuclearXsecBiasingOperator::ProposeOccurenceBiasingOperation(
+            const G4Track* track, const G4BiasingProcessInterface* callingProcess) {
+    
+        /*std::cout << "[ PhotoNuclearXsecBiasingOperator ]: "
+                  << "Parent ID: " << track->GetParentID() 
+                  << " Created within " << track->GetLogicalVolumeAtVertex()->GetName() 
+                  << std::endl;*/
+
+        if (track->GetParentID() != 1) return 0; 
+
+        std::cout << "[ PhotoNuclearXsecBiasingOperator ]: " 
+                  << "Kinetic energy: " << track->GetKineticEnergy() 
+                  << " MeV" << std::endl;
+
+        if (track->GetKineticEnergy() < XsecBiasingOperator::threshold_) return 0; 
+
+        std::cout << "[ PhotoNuclearXsecBiasingOperator ]: " 
+                  << "Calling process: " 
+                  << callingProcess->GetWrappedProcess()->GetProcessName() 
+                  << std::endl;
+
+        std::string currentProcess = callingProcess->GetWrappedProcess()->GetProcessName(); 
+        if (currentProcess.compare(this->getProcessToBias()) == 0) { 
```

Comment:
What happens if the CONVERSION_PROCESS comes before the "ProcessToBias" in the ordered list of possible processes to bias?  E.g. you need pnXsecBiased_ and pnXsecUnbiased_ to bias the CONVERSION_PROCESS?

(this should not matter for this round of simulation)

Reply:
Well, for now, I changed the physics list to force the calling of 'photonNuclear' and 'GammaToMuMu'  before all other gamma processes.  It's easy to change this code such that it forces whatever process that will be biased to be called first.

Reply:
You can see what I'm talking about here: 

https://github.com/LDMXAnalysis/ldmx-sw/pull/418/files#diff-a2ff57790da542c1a46b303f89ec4633